### ISSUE TITLE
Host on the server

### ISSUE THREAD

**Issue by Std1234** [openchatai/OpenChat#55]
Are we allowed to host it on our own server and domain ??

If YES?

any suggestions on where I can deploy this?

**Comment by gharbat** [comment]
Yes you can do that, however, OpenChat is a new project and the current setup is not optimal for production deployment (it's meant for local environments at this early stage).

I'm working on a digital ocean template, so users can deploy openchat on production via 1-click.

**Comment by Std1234** [comment]
> Yes you can do that, however, OpenChat is a new project and the current setup is not optimal for production deployment (it's meant for local environments at this early stage).
> 
> I'm working on a digital ocean template, so users can deploy openchat on production via 1-click.

Yeah, running it locally is very difficult and you have to stick with your PC or laptop, that's why it would be great if you are working on it for 1 click deployment Thanks in advance !!

**Comment by rossman22590** [comment]
+1 for DO install button

**Comment by ingodahn** [comment]
Some instructions how to deploy the openchat_network on my server would be appreciated.
